tests/e2e/framework/testInfo.go

```diff
@@ -141,28 +138,25 @@ func (t testInfo) FetchAndSaveClusterLogs(namespace string) error {
 	if err != nil {
 		return err
 	}
-	lines, err := util.Shell("kubectl get pods -n " + namespace)
+
+	// print table of running pods detailed container information
+	outputFormat := "custom-columns=NAME:.metadata.name,CONTAINERS:.spec.containers[*].name,RESTARTS:.status.containerStatuses[*].restartCount,STATUS:.status.phase"
```

Comment:
This produces the following pod/container summary

```bash
NAME                              CONTAINERS                               RESTARTS   STATUS
details-v1-59f4c7f4dd-spkk9       details,istio-proxy                      0,0        Running
istio-ca-6886f78794-p4ph5         istio-ca                                 0          Running
istio-ingress-54d55b94b5-tgtpz    istio-ingress                            0          Running
istio-mixer-5bf7bb7ddb-bgdmr      statsd-to-prometheus,mixer,istio-proxy   0,0,0      Running
istio-pilot-54c8d6c47-wwbqx       discovery,istio-proxy                    0,0        Running
mongodb-v1-598dfcfbc4-jz6mw       mongodb,istio-proxy                      0,0        Running
productpage-v1-68d9db86d4-hxsw4   productpage,istio-proxy                  0,0        Running
prometheus-cf8456855-lmj8c        prometheus                               0          Running
ratings-v1-8585fffd8d-cm4kh       ratings,istio-proxy                      0,0        Running
ratings-v2-7d96fbf75f-p5lgg       ratings,istio-proxy                      0,0        Running
reviews-v1-86c4987488-hn8gm       reviews,istio-proxy                      0,0        Running
reviews-v2-5cf4dccb6-gr4dv        reviews,istio-proxy                      0,0        Running
reviews-v3-55497db4cb-t9zts       reviews,istio-proxy                      0,0        Running
zipkin-7988c559b7-xpgh6           zipkin                                   0          Running
```

Reply:
if it's easy can you add the image name that will make it easy to spot when somehow we use the wrong image and get into image pull backoff

Reply:
Done